src/components/App.js

```diff
@@ -9,61 +9,69 @@ import yaml from 'js-yaml';
 let githubClient;
 
 export default class App extends Component {
-  constructor(props) {
```

Comment:
The most important style guide ever is beeing consistent. If you contribute
code to a project you should be consistent regardless if you agree with it
or not...

On 8 Sep 2016 12:21 p.m., "Yury Goltsman" <notifications@github.com> wrote:

> In src/components/App.js
> <https://github.com/tikalk/hr-emplyee-profile-manager/pull/9#discussion_r77972405>
> :
>
> > @@ -9,61 +9,69 @@ import yaml from 'js-yaml';
> >  let githubClient;
> >
> >  export default class App extends Component {
> > -  constructor(props) {
>
> Interesting, what is the motivation? Years Web used 4 tabs and boom!!!
> airbnb style changes it to 2 tabs. Lets pray on new standard!!!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tikalk/hr-emplyee-profile-manager/pull/9/files/3352d9b9fb3ff78a27b872b99db3c5a0bb7666e5#r77972405>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAlrRn6v_e0en3BfRJgxK9aNCrB7au11ks5qn9OYgaJpZM4J3DQu>
> .
>